docs/source/guides/getting_started.rst

```diff
@@ -327,9 +329,25 @@ The next chunk of code
 We pull in the properties from the ``build-info.properties`` and make them available to the service
 and more importantly the healthcheck.
 
-In the next binding :java:`bind(HelloWorldResource.class).to(HelloWorldResourceImpl.class);` we are
+AwesomeThingModule
+++++++++++++++++++
+
+.. code-block:: java
+
+  public class AwesomeThingModule extends PrivateModule {
+    protected void configure() {
+      bind(HelloWorldResource.class).to(HelloWorldResourceImpl.class);
+
+      expose(HelloWorldResource.class);
+    }
+  }
+
+``AwesomeThingModule`` is a private module, which contains configuration information for our service. Only the configuration that is explicitly exposed will be available to other modules.
```

Comment:
I'd like to see some information about why we are suggesting using `PrivateModule` instead of `AbstractModule`. Something about how exposing everything from Guice globally can cause dependency issues so it's better to encapsulate groupings of Guice configuration within private modules and only expose the pieces that are necessary for larger/parent groupings. In this case only things needed for JAX-RS/Resteasy should be exposed; including resource classes, providers and features.